.github/workflows/build.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build RPMs
```

Comment:
Can you please move the build automation script into a separate pull request?  This is a useful contribution, though I do not see any direct relation to the other changes.

Reply:
sorry, it's the consequence of working on master. sigh. I'll see if I can remove it from this pr. I should have squashed, too, but I got a bit carried away.

Reply:
hmmm. rebasing seems premature. perhaps  best is if you look at this and when you've had time I'll restructure the PR into multiple ones. I'm also working on docs/interactive help/completion.

Reply:
> Can you please move the build automation script into a separate pull request? This is a useful contribution, though I do not see any direct relation to the other changes.

Ah, I reverted it  and ```git push origin HEAD:master --force``` sufficed.